### ISSUE TITLE
Pip install fails with FileNotFoundError

### ISSUE THREAD

**Issue by zerocewl** [asweigart/pyscreeze#37]
Pip install fails with FileNotFoundError 

`FileNotFoundError: [Errno 2] No such file or directory: 'README.md'`

The error comes up for version pyscreeze>=0.1.15.
Version pyscreeze<=0.1.14 works fine!

Workaround: 

`pip install pyscreeze==0.1.14`

Full stacktrace:

```
$ pip install pyscreeze
Looking in indexes: https://pypi.org/simple
Collecting pyscreeze
  Using cached https://files.pythonhosted.org/packages/c9/6e/e175fda37d76d7846a6cda8a4364dfd7c2d343139b0716c4f03194d14933/PyScreeze-0.1.17.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-z0taorod/pyscreeze/setup.py", line 10, in <module>
        with open("README.md", "r") as fh:
    FileNotFoundError: [Errno 2] No such file or directory: 'README.md'
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-install-z0taorod/pyscreeze/
```

**Comment by Munchotaur** [comment]
Same problem here

**Comment by asweigart** [comment]
Fixed by ec4d181551ebfa081ec635afeef39a59851b0d09

Sorry about that. The MANIFEST.in file didn't include the README.md file, which is now used by setup.py for the long_description field, which I needed so htat PyPI would have a nice-looking description of the module.

**Comment by zerocewl** [comment]
Yeah, seems to be fixed with version 0.1.18.
